packages/block-library/src/missing/edit.native.js

```diff
@@ -31,6 +31,7 @@ export class UnsupportedBlockEdit extends Component {
 		super( props );
 		this.state = { showHelp: false };
 		this.toggleSheet = this.toggleSheet.bind( this );
+		this.closeSheet = this.closeSheet.bind( this );
```

Comment:
Since we've now added a `closeSheet`, it seems like we don't need toggle functionality anymore. Everywhere I see `toggleSheet` being called, it's either to open or close the sheet (never to toggle).
So should `toggleSheet` be renamed to `openSheet` (and its usage updated appropriately to either `openSheet` or `closeSheet`)?

Reply:
Great suggestion! 
but like we agreed over slack, it can be done in the future as it will open a lot of scenarios of testing and we want to ship it with `1.39.0`